components/org.wso2.carbon.identity.oauth.dcr/src/main/java/org/wso2/carbon/identity/oauth/dcr/util/DCRMUtils.java

```diff
@@ -40,6 +40,8 @@
     private static final Log log = LogFactory.getLog(DCRMUtils.class);
     private static Pattern spNameRegexPattern = null;
     private static final String SERVICE_PROVIDERS_NAME_REGEX = "ServiceProviders.SPNameRegex";
+    private static final String APPLICATION_ROLE_PERMISSION_REQUIRED =
+            "OAuth.DCRM.ApplicationRolePermissionRequiredToView";
```

Comment:
Can't we have this in single line?

Reply:
If I change this to a single line, it will exceed the character count for a line.